week-03/main.js

```diff
@@ -12,25 +13,37 @@ var list = [1, 2, 3, 4, 5];
 // `isAlive` - a boolean,
 // and `hobbies` an array.
 // Assign the object to the variable `user`
-
+user = {
+  name: 'Sam',
+  isAlive: true,
+  hobbies: ['running', 'eating'];
+}
 
 
 // Add 'golf' to the `user`'s `hobby` array.
-
+user.hobbies.push('golf');
 
 // # Functions
 
 // Define a function `sayHello` that takes a name string and returns "Hello" plus that name and an exclamation point.
 // Example:
 
 // sayHello('Mer') // "Hello Mer!"
+var sayHello = function(name) {
+  var name = 'Sally';
+  return 'Hello ' + name + '!';
+}
 
 // Define a function `createRange` that takes `min` and `max` as parameters and returns an array with all the numbers
 // between min and max
 // Example:
 
 // createRange(4, 8) // [4, 5, 6, 7, 8]
-
+createRange = function (min, max) {
+  for(var i = min; i < max; i++) {
+    return i;
```

Comment:
Return breaks out of the function. How would you capture `i` at each value and then return all of them as an array?